### PR TITLE
fix(hcl2json): bundle wasm_exec used for building Go instead of looking for it at runtime

### LINKED ISSUE #1865: Project build error
Hello,
I'm trying to build the project, by following the procedure in README.md, but faced on error.

### Steps to Reproduce
```
lerna ERR! yarn run build stdout:
$ tsc
$ node ./lib/__tests__/edge-provider-schema/cli.js ./edge-provider-bindings
The attribute isn't implemented yet: {"storageName":"_boolList","_name":"boolList","type":{"_type":"boolean","typeName":"boolean","isList":true,"isSet":false,"isMap":false,"isComputed":false,"isOptional":false,"isRequired":true,"isSingleItem":false,"level":2,"isNested":false},"optional":false,"computed":false,"terraformName":"boolList","terraformFullName":"edge.required_attribute_resource.boolList","provider":false,"required":true}
The attribute isn't implemented yet: {"storageName":"_boolList","_name":"boolList","type":{"_type":"boolean","typeName":"boolean","isList":true,"isSet":false,"isMap":false,"isComputed":false,"isOptional":true,"isRequired":false,"isSingleItem":false,"level":2,"isNested":false},"optional":true,"computed":false,"terraformName":"boolList","terraformFullName":"edge.optional_attribute_resource.boolList","provider":false,"required":false}
The attribute isn't implemented yet: {"storageName":"_boolList","_name":"boolList","type":{"_type":"boolean","typeName":"boolean","isList":true,"isSet":false,"isMap":false,"isComputed":true,"isOptional":true,"isRequired":false,"isSingleItem":false,"level":2,"isNested":false},"optional":true,"computed":true,"terraformName":"boolList","terraformFullName":"edge.optional_computed_attribute_resource.boolList","provider":false,"required":false}
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
lerna ERR! yarn run build stderr:
node:internal/process/promises:279
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[LinkError: WebAssembly.instantiate(): Import #5 module="go" function="runtime.walltime" error: function import requires a callable]
error Command failed with exit code 1.
lerna ERR! yarn run build exited 1 in '@cdktf/provider-generator'
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

### Environment
- OS: MacOS (M1)
- Node: 16.15.1
- Golang: 1.18

### PR DESCRIPTION
**Pull request by ansgarm**
resolves #1865 while addressing regression in #1899 (which required Go to be installed and did not necessarily match Go version used for building

### CODE REVIEW

File: packages/@cdktf/hcl2json/package.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
     "test:ci": "jest --ci",
     "jest-watch": "jest --watch",
     "package": "./package.sh",
-    "dist-clean": "rm -rf dist"
+    "dist-clean": "rm -rf dist",
+    "prebuild": "cp $GOROOT/misc/wasm/wasm_exec.js ./wasm/wasm_exec.js"
```
Comment (DanielMSchmidt) [inline packages/@cdktf/hcl2json/package.json:19]: Nit: Maybe we can wrap this in a script and throw an error if the goroot is not set

**Review by DanielMSchmidt — APPROVED** [review]

**Comment by ansgarm** [comment]
Oh, oh: Seems to actually fail here: https://github.com/hashicorp/terraform-cdk/runs/7501930280?check_suite_focus=true#step:7:48

File: packages/@cdktf/hcl2json/package.json (hunk 1 above)
Comment (ansgarm) [reply packages/@cdktf/hcl2json/package.json]: done

File: packages/@cdktf/hcl2json/prebuild.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```
Comment (DanielMSchmidt) [inline packages/@cdktf/hcl2json/prebuild.sh:1]: neat :)

**Review by DanielMSchmidt — APPROVED** [review]
🚀

**Pull request merged**